docs/section6/Rule6-2.md

```diff
@@ -14,49 +14,47 @@
 **Manual Check:** None  
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** Table G3.7 and Table G3.8  
-## Rule Logic: 
-
-- For each building_segment in the Proposed Model: ```building_segment_proposed in P_RMR.building.building_segments```  
+**Function Call:** None
 
-- Get the lighting_building_area_type for the building_segment: ```lighting_building_area_type_proposed = building_segment_proposed.lighting_building_area_type```  
+## Rule Logic: 
 
-  - Determine if the building segment uses Building Area Method: ```if lighting_building_area_type_proposed is "None": bam_flag = FALSE else bam_flag = TRUE```  
+- For each building segment in the Proposed Model: `building_segment_p in P_RMR.building.building_segments`  
 
-- For each thermal_block in building_segment: ```thermal_block_proposed in building_segment_proposed.thermal_blocks:```  
+  - If building segment specifies lighting building area type, get the allowable lighting power density from Table G3-8: `if building_segment_p.lighting_building_area_type in table_G3_8: allowable_LPD_BAM = data_lookup(table_G3_8, building_segment_p.lighting_building_area_type)`  
 
-- For each zone in thermal_block: ```zone_proposed in thermal_block_proposed.zones:```  
+  - For each thermal block in building segment: `thermal_block_p in building_segment_p.thermal_blocks:`  
+  
+    - For each zone in thermal block: `zone_p in thermal_block_p.zones:`  
 
-- For each space in zone: ```space_proposed in zone_proposed.spaces:```  
+      - For each space in zone: `space_p in zone_p.spaces:`  
 
-  - Get floor_area from space: ```floor_area_proposed = space_proposed.floor_area```  
+        - Add space lighting power to building segment total: `design_lighting_wattage += space_p.interior_lighting.power_per_area * space_p.floor_area`  
 
```

Comment:
@xingcx if I understand correctly, i think this adds space lighting tower to the building segment lighting power?
If yes, I'm not sure if that is correct. i think we would calculate the lighting power allowance for each space based on G3.7 and the remaining area of the building segment based on G3.8. @mkarpman please confirm.

Reply:
I will change the variable name to building_segment_design_lighting_wattage to avoid confusion. 
Right now this rule calculates:
1). building segment total designed lighting power from all spaces (Line 31, I'll change it to cover multiple lighting objects in space)
2). Table 3.7 allowance, based on each space's area and space type. If any space does not specify space.lighting_space_type, Table 3.7 path will not be used. 
That is I'm assuming we are not comparing Table 3.7 total allowance for spaces with space.lighting_space_type specified and comparing Table 3.8 total allowance for spaces with no space.lighting_space_type specified.
@supriyagoel was this your question? 
3). Table 3.8 allowance, based on building_segment total space area and building_segment.lighting_building_area_type.

Reply:
Agreed, except for #3 Table 3.8 path will not be used if building_segment.lighting_building_area_type is not specified. You are already doing it this way, so this is just to complete this summary of the logic.